### PR TITLE
[#1201] Service summary unification

### LINKED ISSUE #1201: Reorganisation service summary view
In this task we need to change logic of service summary.

Now there we have different summary views depending on service type.
The main goal of this ticket is summary step unification and disable dependency of service type.

The main reason is, we want to remove service type from service model, from now we respect only offer type.

Tasks:
- [x] unification of service summary view

### PR DESCRIPTION
**Pull request by goreck888**
Make summary independent of offer type
Fixes #1201

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Please view this file on the master branch, on stable branches it's out of date.
 - Internal server error message (@goreck888)
 - Anonymous user can enter service offer selection section (@mkasztelnik)
 - Service access button (@martaswiatkowska)
+- Service and offer type enum values (@goreck888)
```
Comment (martaswiatkowska) [inline CHANGELOG.md]: This changes are not connected to this pr

File: app/views/services/summaries/show.html.haml [hunk 2]
```diff
@@ -23,14 +23,12 @@
         %dt Voucher ID
         %dd= @project_item.voucher_id
 
-
   = render "projects/details", project: @project_item.project
 
 = content_for :next_button do
-  = link_to t("service.normal.order.title"),
+  = link_to t("service.#{@service&.service_type}.order.title"),
```
Comment (martaswiatkowska) [inline app/views/services/summaries/show.html.haml]: I think it should be @offer.offer_type

Comment (goreck888) [reply app/views/services/summaries/show.html.haml]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (goreck888) [reply CHANGELOG.md]: Done.

File: app/models/project_item.rb [hunk 3]
```diff
@@ -97,12 +97,12 @@ def research_area_is_a_leaf
     errors.add(:research_area_id, "cannot have children") if research_area&.has_children?
   end
 
-  def vaucherable?
+  def voucherable?
```
Comment (mkasztelnik) [inline app/models/project_item.rb]: Please avoid changes not related to the problem you are solving. Such modification should be done in separate PR.

File: app/views/services/summaries/show.html.haml [hunk 4]
```diff
@@ -23,14 +23,12 @@
         %dt Voucher ID
         %dd= @project_item.voucher_id
 
-
   = render "projects/details", project: @project_item.project
 
 = content_for :next_button do
-  = link_to t("service.orderable.order.title"),
+  = link_to t("service.#{@project_item&.offer.offer_type}.order.title"),
```
Comment (mkasztelnik) [inline app/views/services/summaries/show.html.haml]: This line has 2 problems:
  1. `@project_item&.offer` can return nil if @project_item is nil, then an exception will be thrown
  1. such variables should be set in the controller, a view should be logic-less.

File: app/models/project_item.rb (hunk 3 above)
Comment (goreck888) [reply app/models/project_item.rb]: Ok, I can revert these changes.

File: app/views/services/summaries/show.html.haml (hunk 4 above)
Comment (goreck888) [reply app/views/services/summaries/show.html.haml]: Do you mean I forget about `&.offer_type`?
The same thing is in Marta's PR: 
https://github.com/cyfronet-fid/marketplace/pull/1224/files#diff-e32da2486f29a680e2ab6d4305bbe868R3

File: app/models/project_item.rb (hunk 3 above)
Comment (goreck888) [reply app/models/project_item.rb]: Marked as protip for the future :)
